test/test_quantize_script.py

```diff
@@ -0,0 +1,68 @@
+import torch
+import torch.jit
+from torch.jit._recursive import wrap_cpp_module
+from torch.quantization._quantize_script import script_qconfig
+from torch.quantization import default_observer, default_qconfig
+from torch.testing._internal.common_utils import run_tests
+
+from torch.testing._internal.jit_utils import attrs_with_prefix
+from torch.testing._internal.jit_utils import JitTestCase
+
+class TestScript(JitTestCase):
+    def test_insert_observers_dynamic(self):
```

Comment:
let's use `prepare_dynamic` instead of `insert_observers_dynamic` since that is the API we are going to expose to user